pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -27,13 +27,12 @@ interface IAnalysisWizard {
 export interface IFormValues {
```

Comment:
Update to IAnalysisWizardFormValues so we can reuse this export with the context api.